test/integration/user_install/controls/user_install.rb

```diff
@@ -9,7 +9,7 @@
   describe bash('sudo -H -u vagrant bash -c "source /etc/profile.d/rbenv.sh && rbenv global"') do
     its('exit_status') { should eq 0 }
     its('stdout') { should include(global_ruby) }
-    its('stdout') { should_not match(/system/) }
+    its('stdout') { should_not match(/#{Regexp.quote(system)}/) }
```

Comment:
This should be `#{Regexp.quote("system")}`, making it a  string not an object (that doesn't exist).

Reply:
Failure here 
https://travis-ci.org/sous-chefs/ruby_rbenv/jobs/422229016

Reply:
Good catch. I think I'll close this PR and open another where I replace `match` with `include`, what do you think?

Reply:
I'm happy with that. if it works! :)